### PR TITLE
Facility locator redesign

### PR DESCRIPTION
**Pull request by bshyong**
Implements new designs in this thread: https://github.com/department-of-veterans-affairs/kudos-team/issues/174

screen shot 2016-10-12 at 7 07 18 pm
screen shot 2016-10-12 at 7 05 53 pm
screen shot 2016-10-12 at 6 07 03 pm

### CODE REVIEW

File: src/js/facility-locator/components/SearchControls.jsx [hunk 1]
```diff
@@ -46,20 +46,22 @@ class SearchControls extends Component {
...
+            <input ref="searchField" name="streetCityStateZip" type="text" onChange={this.handleQueryChange} value={currentQuery.searchString}/>
+          </div>
+          <div className="columns medium-5">
+            <label htmlFor="serviceType">Service Type</label>
+            <select name="services" onChange={this.handleFacilityTypeChange} value={currentQuery.serviceType}>
+              <option value="all">All</option>
+              <option value="va_health_facility">Health</option>
+              <option value="va_benefits_facility">Benefits</option>
+              <option value="va_cemetery">Cemeteries</option>
+            </select>
```
Comment (crwallace) [inline src/js/facility-locator/components/SearchControls.jsx]: There should be a "service type" `select` here that has "primary care", "mental health" and "more services coming soon"

**Comment by crwallace** [comment]
For the desktop view we don't want to hide the filtering options ever so we will probably only want to return the filtering inputs.

https://github.com/department-of-veterans-affairs/vets-website/blob/2d81dc25fb38f8e0645fcba10befed088262d637/src/js/facility-locator/components/SearchControls.jsx#L38

File: src/js/facility-locator/components/SearchControls.jsx [hunk 2]
```diff
@@ -46,20 +46,22 @@ class SearchControls extends Component {
...
-          </select>
-          <input type="submit" className="full-width" value="Search" onClick={this.handleSearch}/>
+      <div className="search-controls-container clearfix">
+        <form>
+          <div className="columns medium-5">
+            <label htmlFor="Street, City, State or Zip">Enter Street, City, State or Zip</label>
+            <input ref="searchField" name="streetCityStateZip" type="text" onChange={this.handleQueryChange} value={currentQuery.searchString}/>
+          </div>
+          <div className="columns medium-5">
+            <label htmlFor="serviceType">Service Type</label>
```
Comment (bshyong) [inline src/js/facility-locator/components/SearchControls.jsx]: Renamed

**Comment by crwallace** [comment]
This LGTM! I will write some tests today to make sure it is passing 508.

**Pull request merged**